### PR TITLE
Copy nginx example from staging

### PR DESCRIPTION
**Pull request by mgax**
The nginx config from staging (`/etc/nginx/conf.d/default.conf`) has drifted from the example. This PR updates the example to fit reality.

### CODE REVIEW

**Comment by sorinstelian** [comment]
:+1:

File: config/nginx/nginx.conf.example [hunk 1]
```diff
@@ -55,7 +61,7 @@ server {
     tcp_nopush on;
     gzip on;
     gzip_types text/plain application/x-javascript text/css;
-    expires 1d;
+    add_header Cache-Control "no-store, must-revalidate";
```
[the remark this answers is not in the document]
Comment (melish) [reply config/nginx/nginx.conf.example]: It's related to #1127, until we find a better solution for not caching the "index" after a new release

File: config/nginx/nginx.conf.example [hunk 2]
```diff
@@ -29,6 +29,8 @@ server {
   listen 443 ssl;
   server_name ozone-staging.edw.ro;
 
+  client_max_body_size 0;
```
[the remark this answers is not in the document]
Comment (melish) [reply config/nginx/nginx.conf.example]: In the meantime we have added some uploads done via the django backend ;)

File: config/nginx/nginx.conf.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexkiro) [reply config/nginx/nginx.conf.example]: > It's related to #1127, until we find a better solution for not caching the "index" after a new release

Got it, added a comment there :)

**Pull request merged**